selenium/che-selenium-test/src/main/java/org/eclipse/che/selenium/pageobject/FindUsages.java

```diff
@@ -119,6 +122,7 @@ public void clickOnIconNodeInFindUsagesPanel(String nameNode) {
   public void selectNodeInFindUsagesByDoubleClick(String node) {
     seleniumWebDriverHelper.moveCursorToAndDoubleClick(
         By.xpath(format(Locators.FIND_USAGES_NODE, node)));
+    WaitUtils.sleepQuietly(500, TimeUnit.MILLISECONDS);
```

Comment:
Could you, please, leave a comment in the code with explanation why do we need to wait here for 500 millisecs?